lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -77,7 +79,7 @@ class StartUrl extends Gatherer {
 
   afterPass(options, tracingData) {
     if (!this.startUrl) {
-      return Promise.reject(new Error('No start_url found inside the manifest'));
+      return Promise.reject(ExpectedError(new Error('No start_url found inside the manifest')));
```

Comment:
passing a string with `createExpectedError` has the problem of sentry showing you the line where the error was created and not the more useful line here, and I thought you didn't like things extending from `Error` for some reason but I can't remember why? haha